schemas/questions/calculated.json

```diff
@@ -64,7 +64,7 @@
         "required": [
           "calculated_type",
           "answer_id",
-          "answers_to_group",
+          "answers_to_calculate",
```

Comment:
I think the conditions values above should match our existing condition names
https://github.com/ONSdigital/eq-schema-validator/blob/master/schemas/common_definitions.json#L96

Reply:
@jonnyshaw89 do u mean use the common definitions `conditions` or just the fact that they are a bit more readable i.e. `equal to`, `less than`, `less than or equal to`

Reply:
just use the same values. it doesn't look good where we have multiple ways of writing the same thing

Reply:
but they aren't actually the same thing, they happen to be named similarly, I actually prefer it with the hy[hens in there @nealedj second opinion ?

Reply:
so `equals` isn't the same as `equal-to`?

Reply:
I'm agreeing with Jon here. If **"equal-to"** is checking equality between two things then we have used **"equals"** across the schema. If **"equal-to"** is not checking the equality between two things then the naming is even worse.

Reply:
Also we don't have a concept of **"less-than-equal-to"**. We would specify **"less than"** or **"equals"**.  Allowing multiple conditions to be specified so we have a greater range of possibilities but with less code.